src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1416,6 +1418,61 @@ def test_aks_nodepool_abort(self, resource_group, resource_group_location):
             self.check('powerState.code', 'Running')
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_machine_cmds(self, resource_group, resource_group_location):
```

Comment:
Queued [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=79878517&view=results) to validate the change.